play-json/shared/src/test/scala/play/api/libs/json/JavaTestsEnumsFormat.scala

```diff
@@ -0,0 +1,6 @@
+package play.api.libs.json
+
+object JavaTestsEnumsFormat {
```

Comment:
With separate?

Reply:
What do you mean ?
I looked how the other enum tests are working

"EnumFormat" should {
    import TestEnums.EnumWithCustomNames._

Reply:
There is no need to have a separate file, whereas there are existings file for format tests